packages/ui/src/components/vuestic-components/va-dropdown/VaDropdown.vue

```diff
@@ -130,13 +130,21 @@ export default class VaDropdown extends mixins(DropdownPropsMixin) {
     })
   }
 
-  onAnchorClick (): void {
-    this.$emit('anchor-click')
+  onElementClick (element: string): void {
     if (this.disabled) {
       return
     }
-    if (this.isClicked && !this.closeOnAnchorClick) {
-      return
+    if (element === 'anchor-click') {
+      this.$emit(element)
+      if (this.isClicked && !this.closeOnAnchorClick) {
+        return
+      }
+    }
+    if (element === 'dropdown-click') {
+      this.$emit(element)
+      if (this.isClicked && !this.closeOnClickInside) {
```

Comment:
why would you check for `isClicked` here?

Reply:
Removed this excessive check in the next commit, thanks!